Mixpanel/MixpanelInstance.swift

```diff
@@ -224,8 +224,11 @@ open class MixpanelInstance: CustomDebugStringConvertible, FlushDelegate, AEDele
     var apiToken = ""
     var superProperties = InternalProperties()
     var eventsQueue = Queue()
+    var copyEventsQueue = Queue();
```

Comment:
can we rename these? `copyEventsQueue` and `copyPeopleQueue` don't tell me much about what they actually are since they're not exactly copies of the queues